src/app/learn/index.html

```diff
@@ -1,2 +1,3 @@
 <app-nav></app-nav>
-<app-learn></app-learn>
\ No newline at end of file
+<app-learn></app-learn>
+<stache-back-to-top offset="200"></stache-back-to-top>
```

Comment:
The implementation and code are great! Next...

- [ ] Add this component to the sidebar and default layouts
- [ ] Create a variable on the wrapper, `showBackToTop` which defaults to `true`
- [ ] Write tests :)